recipe/meta.yaml

```diff
@@ -21,6 +21,10 @@ outputs:
       host:
         - python >=3.8
         - pip
+        - packaging >=20
+        - rich
```

Comment:
FYI we dropped `rich` in PR: https://github.com/conda-forge/setuptools_scm-feedstock/pull/71

As best we can tell [`rich` is an optional runtime dependency]( https://github.com/pypa/setuptools_scm/blob/1ac50c22500bceda22c8188782d7cf8af051452f/pyproject.toml#L57-L59 ) and [not used at build time]( https://github.com/pypa/setuptools_scm/blob/1ac50c22500bceda22c8188782d7cf8af051452f/pyproject.toml#L5-L8 ). So it doesn't appear to be needed here

Also it was presenting issues in the bot arch migration: https://github.com/regro/cf-scripts/issues/2275

Please let us know if you have any questions